.thoth.yaml

```diff
@@ -37,3 +36,6 @@ managers:
         - sesheta
       labels: [bot]
       changelog_file: true
+  - name: thoth_advise
```

Comment:
It should be `thoth-advise` https://github.com/thoth-station/kebechet/blob/fda25bc283feabaaee0d144f2c494517deaef7a1/kebechet/managers/__init__.py#L18